forum/utils.py

```diff
@@ -32,3 +44,93 @@ def handle_proxy_requests(request: HttpRequest, suffix: str, method: str) -> Res
         headers=request_headers,
         timeout=5.0,
     )
+
+
+def validate_thread_and_user(
```

Comment:
can we move this code to `models/model_utils` to place mongodb specific code over there?

Reply:
yes sure, we can.